docroot/themes/humsci/humsci_basic/src/scss/utilities/_caption-credit.scss

```diff
@@ -1,6 +1,61 @@
 .field-media-image {
   + .field-media-image-caption {
     @include hb-caption-credit;
+    padding: hb-calculate-rems(16px) hb-calculate-rems(7px) hb-calculate-rems(8px) hb-calculate-rems(8px);
+    display: flex; //Flex is needed for image caption so the items floating right do not list in reverse order.
+    justify-content: flex-end;
+    flex-wrap: wrap;
+
+    @include grid-media-min('sm') { // the right side calculation is lower to adjust for the margin-right on the child elements
+      padding: hb-calculate-rems(24px) hb-calculate-rems(15px) hb-calculate-rems(16px) hb-calculate-rems(12px);
+    }
+
+    h1,
+    h2,
+    h3,
+    h4,
+    h5,
+    h6,
+    blockquote,
+    cite,
+    code,
+    li,
+    dd,
+    dt,
+    span,
+    strong,
+    em,
+    p,
+    .hs-button,
+    .hs-button--big,
+    .hs-secondary-button,
+    .hs-more-link,
+    .hs-external-link,
+    .hs-file-link,
+    .hs-pdf-link,
+    .hs-private-link {
+      display: inline-block;
+    }
+
+    code,
+    h1,
+    h2,
+    h3,
+    .hs-font-splash {
```

Comment:
Added Splash Font to the font-styles override.

Reply:
Thank you!! 🏆